### ISSUE TITLE
Middle click to close a tab deletes the original link (Bug report) 

### ISSUE THREAD

**Issue by Potjoe-97** [TriliumNext/Trilium#3118]
### Trilium Version

0.54.3

### What operating system are you using?

Other Linux

### What is your setup?

Local + server sync

### Operating System Version

Pop_OS! 22.04

### Description

When closing a tab with middle click, the original link followed to open it in the first place is deleted.

Steps to reproduce : 
1) From note "A", follow a link to a note "B" and open it in a new tab. 
2) Move the cursor to the note "B" tab and middle click to close it.
3) In note "A", the original link was deleted. 

Screen capture : 
[Link deleted when closing a tab with middle click.webm](https://user-images.githubusercontent.com/42873357/188607535-607de432-0868-4657-a1fa-427e7bac4462.webm)

**Comment by zadam** [comment]
Hello, that is indeed very strange behavior. Unfortunately I can't reproduce it - the link is not deleted when I do exactly what you did. I wonder if it's perhaps something window manager or mouse related, perhaps middle click sending some additional event or something ...

**Comment by Nriver** [comment]
I've met the same problem. https://github.com/zadam/trilium/issues/2823

It's not like a easy one to solve.

**Comment by Potjoe-97** [comment]
Indeed, I had the occasion to test with the same procedure in windows (I daily drive pop_os!), and the issue is not present.

**Comment by QuentinLeCaignec** [comment]
Same problem here, very strange

**Comment by hatwearingdumb** [comment]
I can reproduce this on Fedora 40.
